private-templates-service/client/src/store/currentTemplate/actions.ts

```diff
@@ -110,6 +110,11 @@ export function updateTemplate(templateID?: string, templateJSON?: string, sampl
 
     let newTemplate = new PostedTemplate();
     const id = templateID || appState.currentTemplate.templateID;
+    
+    const version = currentVersion ? currentVersion :
```

Comment:
Should this default update the latest version instead of the oldest version?

Reply:
good catch